code/datums/jobs/job/xenomorph.dm

```diff
@@ -38,6 +38,11 @@
 	var/datum/hive_status/normal/HS = GLOB.hive_datums[XENO_HIVE_NORMAL]
 	HS.give_larva_to_next_in_queue()
 
+/datum/job/xenomorph/after_spawn(mob/living/L, mob/M, latejoin)
```

Comment:
If your just casting this straight to xenomorph, why is it not already that, would you not avoid the extra cast.
```suggestion
/datum/job/xenomorph/after_spawn(mob/living/carbon/xenomorph/xeno, mob/M, latejoin)
```